### PR TITLE
please let me add my personal overlay

### PR DESCRIPTION
**Pull request by pholthau**

### CODE REVIEW

File: files/overlays/repositories.xml [hunk 1]
```diff
@@ -3515,6 +3515,18 @@ FIN
     <source type="git">https://github.com/perfect7gentleman/pg_overlay.git</source>
     <feed>https://github.com/perfect7gentleman/pg_overlay/commits/master.atom</feed>
   </repo>
+  <repo quality="experimental" status="unofficial">
+    <name>pholthaus-overlay</name>
+    <description lang="en">pholthaus' personal overlay</description>
+    <homepage>https://github.com/pholthau/pholthaus-overlay</homepage>
+    <owner type="person">
+      <email>Patrick Holthaus</email>
```
Comment (mgorny) [inline files/overlays/repositories.xml]: You've got it the other way around.

Comment (pholthau) [reply files/overlays/repositories.xml]: Oops, sorry. Should be fixed now.

**Comment by mgorny** [comment]
Thanks!

**Comment by pholthau** [comment]
Thank you too!

**Pull request closed without merging**